docs/node/statesync.mdx

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 5
+sidebar_label: "Statesync"
+id: "Statesync"
+title: "Using State Sync"
+description: "Learn how to bootstrap a new node using statesync"
+slug: /node/statesync
+---
+
+A new node joining the network needs to catchup with the existing nodes to be able to participate in the network. The default approach the node uses to catch up with the network is to replay all the blocks from the genesis. This process can take a long time depending on the number of blocks in the chain. Statesync is a feature that allows a new node to bootstrap directly from a snapshot instead of replaying all the blocks from the genesis. This rapidly reduces the time it takes to sync a new node to the network, but the node will have a truncated history starting from the height of the snapshot.
```

Comment:
"...rapidly reduces" describes a rate of change for reduction. Maybe just "greatly reduces"?